packages/ui/src/components/preview/index.tsx

```diff
@@ -10,8 +10,7 @@ export const Preview = () => {
   const output = useSelector(outputSelector);
 
   return (
-    // Note we are explicitly setting the font-family to 'initial' here because otherwise the value might leak into the preview from a higher level
-    <div id="preview" style={{ fontFamily: 'initial' }}>
```

Comment:
We can't change this, otherwise the font family of the preview will be overriden

Reply:
This is handled with `all: unset` in https://github.com/tokens-studio/graph-engine/pull/120/file/packages/ui/src/pages/styles.css#diff-c7aa0948f217c055fd253796074b2a7006fffebbbfdee80989d4569a4079060eR101